index.html

```diff
@@ -250,5 +250,25 @@ <h2> About
                 </uli>  
             </div>     
         </article >  
+
+        <article class="contact">
+           
+           <h2 class="head-contact"> Contact me </h2>
+           <p class="contact_p"> 
+            If you have an application you are
+             interested in developing, a feature that
+              you need built or a project that needs 
+              coding. I’d love to help with it 
+           </p>
+
+           <form action="https://formspree.io/f/{form_id}" method="post" >
+            <input type="text" id="name" name="name" placeholder="Your Name" maxlength="30" size="30" required class="in-put"><br>
+            <input name="Email" id="email" type="email" placeholder="E-mail" size="30" required class="in-put"><br>
+
+            <textarea name="comment" form="usrform" cols="30" rows="10" maxlength="500" required class="mess" >Write your message here</textarea>
```

Comment:
- [ ] Nice implementation of `textarea`. but there is one more thing if you rectify it, it will make your form looks nice. Please use `placeholder =" Write your message here" `instead of a plain text.